swap_meet/item.py

```
@@ -1,2 +1,34 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=None):
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+        self.age = age
+    
+        if isinstance(self.id, str):
+            raise ValueError("IDs need to be numerical")
```

Nice consideration to prevent creating an Item with a string id! This will prevent a string from being passed in; what will happen if someone passes an invalid data type like a `list` in for the `id` parameter? 
- Is there a way we could raise this error if the `id` is not an integer?